src/hotspot/cpu/aarch64/aarch64_neon.ad

```diff
@@ -4243,32 +4243,64 @@ instruct vxor16B(vecX dst, vecX src1, vecX src2)
 
 // ------------------------------ Shift ---------------------------------------
 
-instruct vshiftcnt8B(vecD dst, iRegIorL2I cnt) %{
+// Vector shift count
+// Note-1: Low 8 bits of each element are used, so it doesn't matter if we
+//         treat it as ints or bytes here.
+// Note-2: Shift value is negated for RShiftCntV additionally. See the comments
+//         on vsra8B rule for more details.
+
+instruct vslcnt8B(vecD dst, iRegIorL2I cnt) %{
   predicate(n->as_Vector()->length_in_bytes() == 4 ||
-            n->as_Vector()->length_in_bytes() == 8);
+                            n->as_Vector()->length_in_bytes() == 8);
```

Comment:
We may improve the style here, i.e. removing the leading whitespace.

Reply:
Not in a backport.

Reply:
I see. Thanks for your comment.